example_pkg/meson.build

```
@@ -3,7 +3,7 @@ project(
   'c',
   version: '0.0.dev0',
   license: 'BSD-3',
-  meson_version: '>= 0.64.0',
+  meson_version: '>= 1.4.0',
```

```suggestion
  meson_version: '>= 1.61.0',
```

Testing to see if we can get CodeQL to run on this repo. It uses an older version of meson.